src/sample_package/src/MyNode.cpp

```diff
@@ -14,6 +14,11 @@ MyClass::MyClass(int argc, char **argv, std::string node_name) {
     ros::NodeHandle nh;
     ros::NodeHandle public_nh("~");
 
+    // Obtains character from the parameter server (or launch file), sets '!' as default
+    std::string parameter_name = "my_node/character";
+    std::string default_character = "!";
+    nh.param(parameter_name, suffix, default_character);
```

Comment:
We should be using `SB_getParam` here, from the kitchen sink library :wrench: . I've already added all the stuff to the cmake, should just be a matter of using it. Example [here](https://github.com/UBC-Snowbots/IGVC-2017/blob/master/src/decision/src/LidarDecision.cpp)